Source/Scene/Globe.js

```diff
@@ -672,7 +692,7 @@ Globe.prototype.pickWorldCoordinates = function (
       );
     } else if (defined(surfaceTile.renderedMesh)) {
       BoundingSphere.clone(
-        surfaceTile.renderedMesh.boundingSphere3D,
+        surfaceTile.tileBoundingRegion.boundingSphere,
```

Comment:
The mesh's `boundingSphere3D` is not exaggerated, so it can't be used. The `TileBoundingRegion.boundingSphere` has the updated bounding volumes.